h/static/scripts/controllers.coffee

```diff
@@ -118,6 +121,9 @@ class AppController
     $scope.clearSelection = ->
       $scope.search.query = ''
       annotationUI.clearSelectedAnnotations()
+      crossframe.notify
```

Comment:
@nickstenning, how about introducing a wrapper service (something like `AnnotationSelectionService`), which would provide a public API for defining/changing the selection, and having this intermediary service talk to both `AnnotatorUI`, and make the cross-frame calls?